docs/reference/indices/index-templates.asciidoc

```diff
@@ -29,6 +23,10 @@ specify settings, mappings, and aliases.
 
 If a new data stream or index matches more than one index template, the index template with the highest priority is used.
 
+When a composable template matches a given index
+it always takes precedence over a legacy template. If no composable template matches, a legacy
+template may still match and be applied.
+
 If an index is created with explicit settings and also matches an index template,
 the settings from the create index request take precedence over settings specified in the index template and its component templates.
 
```

Comment:
It may be helpful to add some callouts to the following snippet, particularly for the `composed_of` param.

Reply:
I'm going to leave this and other content changes for the next pass.